src/libraries/System.Private.Xml/tests/TrimmingTests/XslCompiledTransformTests.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Xml.XPath;
+using System.Xml.Xsl;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        using (StringReader text = new StringReader(
+    @"<?xml version=""1.0"" encoding=""UTF-8""?>
+<catalog>
+    <cd>
+        <title>Empire Burlesque</title>
+        <artist>Bob Dylan</artist>
+        <country>USA</country>
+        <company>Columbia</company>
+        <price>10.90</price>
+        <year>1985</year>
+    </cd>
+</catalog>"))
+        {
+
+
+            XPathDocument myXPathDoc = new XPathDocument(text);
+            XslCompiledTransform myXslTrans = new XslCompiledTransform();
+            string xmlStr =
+@"<?xml version=""1.0""?>
+<xsl:stylesheet version=""1.0"" xmlns:xsl=""http://www.w3.org/1999/XSL/Transform"">
+<xsl:template match=""/"">
+<html>
+<body>
+<h2> My CD Collection</h2>
+   <table border = ""1"">
+      <tr bgcolor = ""#9acd32"">
+         <th> Title </th>
+         <th> Artist </th>
+       </tr>
+
+      <xsl:for-each select = ""catalog/cd"">
+          <tr>
+            <td><xsl:value-of select = ""title"" /></td>
```

Comment:
Nit: I understand this test only for showing if trimming works, just wondering if the updates made in `XmlIlVisitor.cs` are covered in the unit tests somewhere

Reply:
It looks like there are a bunch of XslCompiledTransform tests here:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XslCompiledTransform.cs